Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -40,16 +44,25 @@ YUI.add('ez-contenteditviewservice', function (Y) {
         _load: function (next) {
             var request = this.get('request'),
                 service = this,
-                languageCode = this.get('languageCode');
+                languageCode = this.get('languageCode'),
+                baseLanguageCode = this.get('baseLanguageCode'),
+                languageCodeForLoadContent = baseLanguageCode ? baseLanguageCode : languageCode;
 
             this.get('version').reset();
-            this._loadContent(request.params.id, languageCode, function () {
+            this._loadContent(request.params.id, languageCodeForLoadContent, function () {
                 var tasks,
-                    version = service.get('version'),
                     content = service.get('content'),
+                    translationExists,
                     resources;
 
-                version.set('fields', content.get('fields'));
+                translationExists = service._checkIfTranslationExists(content, languageCodeForLoadContent);
+
+                if (baseLanguageCode && !translationExists) {
+                    service._error(
+                        "Could not load the content with id '" + content.get('contentId')
+                        + "' and languageCode '" + baseLanguageCode + "'"
+                    );
```

Comment:
I guess you should add a `return` here as it's useless to try to load the owner, location and content type